lib/mayaUsd/render/vp2RenderDelegate/meshViewportCompute.cpp

```diff
@@ -22,14 +22,15 @@
 #include <mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h>
 
 #include <pxr/imaging/hd/vertexAdjacency.h>
-#include <pxr/imaging/pxOsd/refinerFactory.h>
-#include <pxr/imaging/pxOsd/tokens.h>
 #include <pxr/pxr.h>
 
 #include <maya/MProfiler.h>
 
 #ifdef HDVP2_ENABLE_GPU_COMPUTE
 
+#include <pxr/imaging/pxOsd/refinerFactory.h>
+#include <pxr/imaging/pxOsd/tokens.h>
```

Comment:
I thought moving these includes here would break the clang-format (regarding the rules about include order). But the preflight passed. Is ```HDVP2_ENABLE_GPU_COMPUTE``` removing the entire file? Can that line (29) just be moved up after ```#include "meshViewportCompute.h"``` and leave these two headers where they were?

Reply:
I have updated the code: [`6d27610`](https://github.com/Autodesk/maya-usd/pull/1723/commits/6b27610d5ada2b757461678b26c4e22e0999aea5)

Reply:
Great thanks.